.travis.yml

```diff
@@ -142,12 +131,31 @@ matrix:
             - libboost-log1.55-dev
             - ninja-build
 
+
 before_script:
-  - ./install_dependencies.sh
-  - export PATH=$PATH:$PWD/odb/odb-2.4.0-x86_64-linux-gnu/bin/
-  - cmake -DBUILD_STATIC=1 -DCMAKE_CXX_COMPILER=${CXX_COMPILER} -DCMAKE_C_COMPILER=${C_COMPILER} -DCMAKE_CXX_FLAGS="-Iodb/libodb-2.4.0 -Iodb/libodb-sqlite-2.4.0" -DCMAKE_EXE_LINKER_FLAGS="-Wl,-rpath,odb/ -Lodb/"  -DODB_PATH=$PWD/odb/ -G Ninja
+  - |
+    if [[ $TRAVIS_OS_NAME == "linux" ]]; then
+      ./install_dependencies.sh
+      export PATH=$PATH:$PWD/linux_dependencies/odb-2.4.0-x86_64-linux-gnu/bin/
+    fi
+  - |
+    if [[ $TRAVIS_OS_NAME == "osx" ]]; then
+      ./install_dependencies.sh osx
+      export PATH=$PATH:$PWD/osx_dependencies/odb-2.4.0-i686-macosx/bin/
+    fi
+
 
 script:
-  - ninja -j 1 -v
-  - ninja test
+  - |
+    if [[ $TRAVIS_OS_NAME == "linux" ]]; then
+      echo "running cmake"
```

Comment:
The Travis log already lists when the cmake command is executed, so this echo can be removed.